### DISCUSSION TITLE
new Intel vulnerability: Load Value Injection (CVE-2020-0551 / INTEL-SA-00334)

### DISCUSSION THREAD

**Discussion by lilyanatia** [speed47/spectre-meltdown-checker#544] [Other vulnerabilities / information]
https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00334.html

https://software.intel.com/security-software-guidance/software-guidance/load-value-injection

**Comment by speed47** [comment]
Key takeaways:
This new vuln will apparently not get any kernel mitigation, and Intel clearly stated that there was no microcode update planned.
It seems to be very difficult to run this attack in a non-lab environment, and systems mitigated against Spectre/L1TF/MDS have additional protections that make it yet more difficult.
On the SGX side, Intel has released a new SDK with additional measures to block this attack from enclaves.
Keeping this open just for information.

**Comment by speed47** [comment]
Intel never changed their stance on this one, no microcode and no mitigation: there's nothing to check for. Listed in the FAQ for documentation purposes: https://github.com/speed47/spectre-meltdown-checker/blob/master/FAQ.md#which-rules-are-governing-the-support-of-a-cve-in-this-tool
